lib/rules/jsx-no-literals.js

```diff
@@ -40,27 +40,49 @@ module.exports = {
         },
         ignoreProps: {
           type: 'boolean'
+        },
+        noAttributeStrings: {
+          type: 'boolean'
         }
       },
+
```

Comment:
Remove unnecessary new line

Reply:
+1, altho if it helps readability it'd be fine

Reply:
New line removed.